streams/src/test/java/org/apache/kafka/streams/tests/StreamsOptimizedTest.java

```diff
@@ -130,6 +130,8 @@ public static void main(final String[] args) throws Exception {
             }
         });
 
+        if (streamsProperties.containsKey("streams.cleanup")
+            && Boolean.parseBoolean(streamsProperties.getProperty("streams.cleanup"))) streams.cleanUp();
```

Comment:
I think we can just call `cleanUp` here without passing in a flag. We only _need_ to do it the second time around, but it doesn't hurt to just call it every time and it keeps things simple (the system tests are already complicated enough 🙂)

Reply:
Copy that. one line change for this PR :)

Reply:
One liners are the best 👍